src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -295,15 +299,19 @@
     margin-left: 4px;
 }
 
-.sf-toolbar-block:hover {
+.sf-toolbar-block:hover,
+.sf-toolbar-block.hover {
     position: relative;
 }
-.sf-toolbar-block:hover .sf-toolbar-icon {
+.sf-toolbar-block:hover .sf-toolbar-icon,
+.sf-toolbar-block.hover .sf-toolbar-icon
+{
```

Comment:
this curly brace should stay on previous line for consistency with all other rules